SimpleCalculator/SimpleCalculator/Program.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+
+namespace SimpleCalculator
+{
+    
+    public enum Operations
+    {
+        Multiplication = 1,
+        Division,
+        Addition,
+        Subtraction,
+        Exponentiation,
+        Factorial
+    }
+
+                          
+    class Program
+    {
+        public static double Calc(Operations operation, double operand1, double operand2 = 0)
+        {
+            var result = 1d;
+
+            switch (operation)
+            {
+                case Operations.Multiplication: result = operand1 * operand2;
+                    break;
+                case Operations.Division: result = operand1 / operand2;
```

Comment:
What about cheking 2-nd operand to zero equal to avoid dividing by zero? (як на рахунок перевірки другого операнда на рівність нулю, щоб уникнути ділення на нуль?) // переклав бо сам не впевнений шо англійською коректно написав

Reply:
Solved in the last commit.